### PR TITLE
chore(documentation-v7): add stories for all doc pages

### PR DESCRIPTION
**Pull request by alizedebray**
All meta are now declared in `.stories.ts` files so that badges are displayed correctly.

### CODE REVIEW

File: packages/documentation-v7/.storybook/preview.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ const preview: Preview = {
           'Templates',
           'Utilities',
           'Misc',
-          ['Migration Guide', 'ChangeLog', 'Versions'],
+          ['Migration Guide', 'Change Log', 'Versions'],
```
Comment (imagoiq) [inline packages/documentation-v7/.storybook/preview.ts:45]: I'm more used to Changelog. I think both are right though, not sure.

**Review by imagoiq — APPROVED** [review]

**Review by gfellerph — CHANGES_REQUESTED** [review]
There is an issue with importing markdown as raw https://github.com/swisspost/design-system/actions/runs/5834611361/job/15824527873?pr=1788#step:5:97. Maybe it's due to using vite instead of webpack with storybook v7.

File: packages/documentation-v7/.storybook/preview.ts (hunk 1 above)
Comment (alizedebray) [reply packages/documentation-v7/.storybook/preview.ts]: I renamed "Change Log" to "Changelog"

**Comment by alizedebray** [comment]
> There is an issue with importing markdown as raw https://github.com/swisspost/design-system/actions/runs/5834611361/job/15824527873?pr=1788#step:5:97. Maybe it's due to using vite instead of webpack with storybook v7.

The URL wasn't correct, it should be okay now.

**Comment by alizedebray** [comment]
> There is an issue with importing markdown as raw https://github.com/swisspost/design-system/actions/runs/5834611361/job/15824527873?pr=1788#step:5:97. Maybe it's due to using vite instead of webpack with storybook v7.

The path wasn't correct, I just updated it.

**Review by gfellerph — APPROVED** [review]

**Pull request merged**